source/administration/config-settings.rst

```diff
@@ -3078,6 +3093,18 @@ Specify the color of the AD/LDAP login button for white labeling purposes. Use a
 | This feature's ``config.json`` setting is ``"LoginButtonColor": ""`` with string input.                                       |
 +-------------------------------------------------------------------------------------------------------------------------------+
 
+SAML session lengths
+^^^^^^^^^^^^^^^^^^^^^^
+
+SAML session length is defined per
+
+ - ``url`` : ``environment/session_lengths``
+- ``title`` : ``admin.sidebar.sessionLengths``
+- ``type`` : ``TYPE_NUMBER``
+- ``label`` : ``mobileSessionDays``
+- ``help_text`` : ``admin.service.ssoSessionDaysDesc``
+- ``placeholder``: ``sessionIdleTimeout``
```

Comment:
I'm not sure the above makes sense? How does that relate to SAML session length

Reply:
We had a long discussion and multiple iteration on https://github.com/mattermost/docs/pull/2732, should I remove `placeholder`?

Reply:
The content of this section doesn't make any sense to me. I'm not sure what it's trying to communicate. What does any of it mean? For example, what does the below communicate to a reader of this document?

```
``help_text`` : ``admin.service.ssoSessionDaysDesc``
```

Reply:
The point is, I don't remember now. I have deleted these lines - fa3495e.